device.go

```diff
@@ -76,8 +88,12 @@ type Light struct {
 
 // A jsonLight is the raw JSON representation of a Light.
 type jsonLight struct {
-	On          int `json:"on"`
-	Brightness  int `json:"brightness"`
+	On int `json:"on"`
+
+	// Brigthness has a range of 3-100
```

Comment:
Apologies, I meant to append these limitations to the doc comments on Light since that is exported for the user to see.

Reply:
😝